make_queue/migrations/0023_machine_stream_name.py

```diff
@@ -23,7 +26,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='machine',
             name='stream_name',
-            field=models.CharField(blank=True, default=None, max_length=30, null=True, validators=[RegexValidator(code='NOT_URL_SAFE', message='Stream name can only consist of english letters, numbers, hyphens or underscores.', regex='^[a-zA-Z0-9_-]+$')], verbose_name='Stream Name'),
+            field=models.CharField(blank=True, default='', max_length=50, validators=[django.core.validators.RegexValidator(code='invalid_stream_name', message='This can only consist of lowercase English letters, numbers, hyphens or underscores.', regex=re.compile('^[a-z0-9_-]+$'))], verbose_name='stream name'),
```

Comment:
Does the `message` field need to wrapped inside a translation `_()`?

Reply:
Ops, I just realized this was a migration file. I thought this was on the model 😅